cmd/controller/main.go

```diff
@@ -146,14 +146,23 @@ type controllerRunOptions struct {
 }
 
 func main() {
-	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
-	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	// This is also being registerd in kubevirt.io/kubevirt/pkg/kubecli/kubecli.go so
+	// we have to guard it
+	//TODO: Evaluate alternatives to importing the CLI. Generate our own client? Use a dynamic client?
```

Comment:
Can we push upstream for a change?

Reply:
Opened https://github.com/kubevirt/kubevirt/issues/1793